### PR TITLE
Add test coverage for edit functions with TODOs

### PR DESCRIPTION
**Pull request by leeyspaul**
Adds test coverage for simpler functions in `edit_functions.py`.

Added TODOs for remainder which takes further digging in Anki source code.

### CODE REVIEW

**Comment by leeyspaul** [comment]
👀 @scottgigante

File: tests/test_edit_functions.py [hunk 1]
```diff
@@ -33,7 +73,74 @@ def test_translate_local_multiple_results(
...
+    mwo_일월_second = MockWordObject()
+
+    mwo_밥_first.english = "食"
+    mwo_일월_second.english = "一月"
+
+    mock_translate_local_results = [mwo_밥_first, mwo_일월_second]
+
+    mocked_translate_local.return_value = mock_translate_local_results
+
+    expected_built_str = "食,一月"
```
Comment (scottgigante) [inline tests/test_edit_functions.py:105]: Is this test missing an assert or am I not understanding?

Comment (leeyspaul) [reply tests/test_edit_functions.py]: You are totally right, great catch haha

Comment (leeyspaul) [reply tests/test_edit_functions.py]: I also missed the hanja attributes too. 🤦

Comment (leeyspaul) [reply tests/test_edit_functions.py]: Think I got it this time though!

**Pull request merged**